includes/REST_API/Fonts_Controller.php

```diff
@@ -79,7 +79,8 @@ public function register_routes() {
 	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
 	 */
 	public function get_items( $request ) {
-		$fonts       = Fonts::get_fonts();
+		$font        = new Fonts();
```

Comment:
`$font` is also used in the `foreach` loop below, which is confusing and error prone.

Suggestion:

Rename variables like so:

```php
$fonts = new Fonts();
$fonts_list = $font->get_fonts();
```

I think in the tests it's already like this, so this change would make things consistent.

Reply:
Fixed.